actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -43,6 +43,18 @@ def initialize(params) # :nodoc:
     end
   end
 
+  # Raised when a Parameters instance is not marked as permitted and
+  # an operation to transform to hash is called.
+  #
+  #   params = ActionController::Parameters.new(a: "123", b: "456")
+  #   params.to_h
+  #   # => ActionController::ForbiddenParameters: converting a unpermitted parameters to hash is not allowed
+  class ForbiddenParameters < StandardError
+    def initialize
+      super("converting an unpermitted parameters to hash is not allowed")
```

Comment:
"unable to convert unpermitted parameters to hash", maybe? "not allowed" sounds like we're being difficult.